### PR TITLE
Replace lv_label_set_text with better options where possible

### PR DESCRIPTION
**Pull request by jonvmey**
If the text being set is static use lv_label_set_text_static to avoid
unnecessary memory allocations. Otherwise if a temporary char buffer is
being sprintf-ed and then set to the label used lv_label_set_text_fmt to
avoid the intermediary data copies.

### CODE REVIEW

File: src/displayapp/screens/Music.cpp [hunk 1]
```diff
@@ -212,33 +212,30 @@ bool Music::Refresh() {
...
-            ((totalLength % (60 * 60)) / 60) % 100);
-    lv_label_set_text(txtTrackDuration, timer);
+    lv_label_set_text_fmt(txtTrackDuration,
+                          "%02d:%02d/%02d:%02d",
+                          (currentLength / (60 * 60)) % 100,
+                          ((currentLength % (60 * 60)) / 60) % 100,
+                          (totalLength / (60 * 60)) % 100,
+                          ((totalLength % (60 * 60)) / 60) % 100);
   } else {
     char timer[12];
```
Comment (hatmajster) [inline src/displayapp/screens/Music.cpp:232]: this is probably also obsolete

**Comment by hatmajster** [comment]
I was wondering how it will impact internationalization: https://github.com/JF002/InfiniTime/pull/417 but as all text is static it shouldn't be an issue at any point... I hope

File: src/displayapp/screens/Music.cpp (hunk 1 above)
Comment (jonvmey) [reply src/displayapp/screens/Music.cpp:232]: Yes, thanks for pointing that out. I'll remove it too.

**Comment by Riksu9000** [comment]
@jonvmey Hi. Can you fix the conflicts, so we could merge this?

**Comment by Riksu9000** [comment]
These improvements have been added in #1044. Thanks!

**Pull request closed without merging**